examples/full-screen-source.js

```diff
@@ -0,0 +1,28 @@
+goog.require('ol.Map');
+goog.require('ol.View');
+goog.require('ol.control');
+goog.require('ol.control.FullScreen');
+goog.require('ol.layer.Tile');
+goog.require('ol.source.OSM');
+
+
+var view = new ol.View({
+  center: [-9101767, 2822912],
+  zoom: 14
+});
+
+var map = new ol.Map({
+  controls: ol.control.defaults().extend([
+    new ol.control.FullScreen({
+      fullScreenSource: 'fullscreen'
+    })
```

Comment:
Curious if anybody else has opinions on this.  I think `source` would make a sensible property name (instead of `fullScreenSource`).